### PR TITLE
misc: support dart2 stable, prepare to release 0.1.11

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
It doesn't look like there were breaking changes from 0.1.10...

**Comment by kevmoo** [comment]
@isoos – would you squash/merge and publish ASAP if this looks okay...

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: pwa
-version: 0.2.0
+version: 0.1.11
```
Comment (alorenzen) [inline pubspec.yaml:2]: Should probably also bump the SDK constraint to only support >=2.0. I noticed a change from `JSON` to `json`, which is breaking for dart 1.XX

Comment (kevmoo) [reply pubspec.yaml:2]: ack!

**Pull request merged**